korman/properties/modifiers/sound.py

```diff
@@ -368,6 +372,8 @@ def _convert_sound(self, exporter, so, pClass, wavHeader, dataSize, channel=None
             sound.properties |= plSound.kPropLooping
         if self.incidental:
             sound.properties |= plSound.kPropIncidental
+        if self.local_only:
+            sound.properties |= plSound.kPropIsLocalOnly
```

Comment:
There is an existing `kPropLocalOnly` that should do what you want here: https://github.com/H-uru/libhsplasma/blob/35b100f64d6fe54479737e885d9809fcebafdf86/core/PRP/Audio/plSound.h#L35

Reply:
Ah, so I got the property name wrong. I'll try that and let you know.